source/api_documentation/network_integration/ringpools/index.rst

```diff
@@ -48,10 +51,6 @@ Similar to Promo Numbers, because these are generated by Invoca, the interface i
     - integer
     - The maximum amount of phone numbers to be allotted at one time.
 
-  * - param1
-    - string
-    - Dynamic number pools have attributes named param1 through param10 for custom use by the user.
-
   * - preferred
```

Comment:
Ticket asked about `preferred` versus `is_first_preference`

Looks like "test_click_url", "sample_api_url" and "is_first_preference" are all ignored: https://github.com/Invoca/web/blob/75416e98ed06cc608a605df415c1b0e885e1a2ea/test/unit/api/ring_pool_dto_test.rb#L151-L180

and thus we should remove all three from this table and all examples!

Reply:
+1